.github/workflows/edge-home-orchestration-go.yml

```diff
@@ -38,6 +39,10 @@ jobs:
       - name: Build the container
         run: ./build.sh container ${{ matrix.arch }}
 
+      - name: Analysis using the lint & vet tools
+        if:  ${{ matrix.arch == 'x86_64' }}
```

Comment:
It is not necessary to run the analysis on every OS.

Reply:
I could not give preference to any particular system, so I'm testing on all! )

Reply:
> I could not give preference to any particular system, so I'm testing on all! )

Do with latest version! 😁